modules/rostests/apitests/shell32/CMakeLists.txt

```diff
@@ -67,7 +68,7 @@ add_rostests_file(TARGET shell32_apitest)
 
 # shell32_apitest_sub.exe
 add_executable(shell32_apitest_sub shell32_apitest_sub.cpp)
```

Comment:
The _sub test also needs the cppstl? It seems to compile only one single file shell32_apitest_sub.cpp that you haven't touched.

Reply:
Will double check.